### ISSUE TITLE
Manually specifying config script file or config script directory

### ISSUE THREAD

**Issue by dolandemort** [Blightmud/Blightmud#592]
With TinyFugue, we had the ability to specify a config file to be loaded at command execution instead of using the default `$HOME/.tfrc`.

With Blightmud, unless I have missed something obivious, I cannot find an equivalent ability to override the hardcoded default of the folder `$HOME/.config/blightmud` having all its config files loaded automatically on start.

Is there some way to accomplish this? Maybe I'm weird, but I like to have my MUDs separated into their own folders, and I don't necessarily want to use the hardcoded default folder because of the way I have my partitions/drives setup.

**Comment by LiquidityC** [comment]
That's right. The config dir is always read. But you can still achieve the functionality you need. 


`/help config_scripts` should give you some ideas. Also viewable here: https://github.com/Blightmud/Blightmud/blob/dev/resources/help/config_scripts.md

**Comment by dolandemort** [comment]
Seems to be an acceptable workaround. It's a little awkward having to maintain scripts in two locations instead of one, though. Thanks for the quick response.

Just want to say, in case no one tells you often enough, this is a great project, and I really appreciate you making it available to the rest of us. The scripting syntax of TinyFugue really is awkward in comparison to something with a native language like this.

**Comment by LiquidityC** [comment]
I maintain all my scripts in one workspace. I just symlink the "main" file into `~/.config/blightmud/`. Eg. `ln -s <my-main-script-file> ~/.config/blightmud/`. Keeps it nice and easy to work with. Works better if you have your scripts in a git repo too.

Thanks, I'm glad you like it. I got quite used to tf script back in the day. My main issue with TF was how it fell behind in functionality and there was no centralized official maintained version. Also, I really wanted to make a mud client. :smile:
